src/index.html

```diff
@@ -16,17 +16,25 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="input input__icon input__icon--big">
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try “Los Angeles“"
```

Comment:
The `placeholder` attribute value 'Try “Los Angeles“' includes spaces which are not recommended in links or placeholders that resemble links. Consider removing spaces if this is meant to be a link or ensure the placeholder text does not mislead users into thinking it is a link.